### PR TITLE
act documentation

### PR DESCRIPTION
**Pull request by justelad**
add documentation for the act feature.

resolves https://github.com/ethersphere/go-ethereum/issues/867

### CODE REVIEW

File: contents/usage/act.rst [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+private key, identical to those used throughout Ethereum for accessing accounts. 
+
+In order to obtain the content reference, an
+`Elliptic-curve Diffie–Hellman <https://en.wikipedia.org/wiki/Elliptic-curve_Diffie%E2%80%93Hellman>`_ `(ECDH)` 
+key agreement needs to be performed between a provided EC public key (that of the content publisher) 
+and the authorized key, after which the undisclosed authorized party can decrypt the reference to the 
+access controlled content.
+
+Whether using access control to disclose content to a single party (by using the ``pk`` strategy) or to 
+multiple parties (using the ``act`` strategy), a third unauthorized party cannot assert the identity
```
Comment (zelig) [inline contents/usage/act.rst]: assert? find out? know?

File: contents/usage/act.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+.. danger:: 
+  **NEVER (EVER!) use an external gateway to upload or download access controlled content as you will be putting your privacy at risk!
+  You have been fairly warned!**
+
+Usage
+-----
+
+**Creating** access control for content is currently supported only through CLI usage.
+
+**Accessing** restricted content is available through CLI. Accessing content which is restricted
```
Comment (zelig) [inline contents/usage/act.rst]: why? this is confusing. Accessing content using pk and act strategy are also available through http api.

File: contents/usage/act.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+**Accessing** restricted content is available through CLI. Accessing content which is restricted 
+by a passphrase is also available through the HTTP API 
+using `HTTP Basic access authentication <https://en.wikipedia.org/wiki/Basic_access_authentication>`_.
+
+
+CLI usage
+---------
+
+.. important:: Restricting access to content on Swarm is a 2-step process - you first upload your content, then wrap the reference with an access control manifest. **We recommend that you always upload your content with encryption enabled**. For the sake of the following examples we will refer to this hash as ``REF``
```
Comment (zelig) [inline contents/usage/act.rst]: 'this hash' ?
maybe say explicitly that the returned reference is referred to as REF.

File: contents/usage/act.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+
+**Creating a password protected manifest:**
```
Comment (zelig) [inline contents/usage/act.rst]: this title may be confusing. Maybe:
creating password protected content, Step 1: creating the access manifest

File: contents/usage/act.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+.. code-block:: bash
+
+	$ swarm --bzzaccount 2f1cd699b0bf461dcfbf0098ad8f5587b038f0f1 access new pk --grant-key 02e6f8d5e28faaa899744972bb847b6eb805a160494690c9ee7197ae9f619181db <REF>
+	4b964a75ab19db960c274058695ca4ae21b8e19f03ddf1be482ba3ad3c5b9f9b
+
+The returned hash ``4b964a75ab19db960c274058695ca4ae21b8e19f03ddf1be482ba3ad3c5b9f9b`` is the hash of the access controlled manifest. 
+
+The only way to fetch the access controlled content in this case would be to request the hash through one of the nodes that were granted access and/or posses the granted private key - either the local node that was used to upload the content or the node which was granted access through its public key
+
+**Creating a password protected manifest (multiple recipients):**
```
Comment (zelig) [inline contents/usage/act.rst]: here exlplain again that act is to grant access to multiple parties via their keys and therefore need a list of public keys for the grantees This list is communicated to the CLI by giving a file containing the keys in format X one per line. The file can(not?) contain extra whitespace and comments

File: contents/usage/act.rst [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+.. note:: Grantee public keys are expected to be in an *secp256 compressed* form - 66 characters long string (e.g. ``02e6f8d5e28faaa899744972bb847b6eb805a160494690c9ee7197ae9f619181db``). Each grantee should appear in a separate line
+
+.. code-block:: bash
+
+	$ swarm --bzzaccount 2f1cd699b0bf461dcfbf0098ad8f5587b038f0f1 access new act --grant-keys /path/to/public-keys/file <REF>
+	4b964a75ab19db960c274058695ca4ae21b8e19f03ddf1be482ba3ad3c5b9f9b
+
+The returned hash ``4b964a75ab19db960c274058695ca4ae21b8e19f03ddf1be482ba3ad3c5b9f9b`` is the hash of the access controlled manifest. 
+
+As with the ``pk`` strategy - the only way to fetch the access controlled content in this case would be to request the hash through one of the nodes that were granted access and/or posses the granted private key - either the local node that was used to upload the content or one of the nodes which were granted access through their public keys.
```
Comment (zelig) [inline contents/usage/act.rst]: it is not enough for the node to 'possess' the private key. It needs to have started with the key as your bzz-account

File: contents/usage/act.rst (hunk 2 above)
Comment (justelad) [reply contents/usage/act.rst]: true. my mistake. this is now corrected.

File: contents/usage/act.rst (hunk 4 above)
Comment (justelad) [reply contents/usage/act.rst]: reworded this. have a look if its better. same for the others below.

**Pull request merged**